core/src/net/sf/openrocket/simulation/FlightData.java

```diff
@@ -310,4 +308,42 @@ private double calculateMaxAcceleration() {
 		
 		return max;
 	}
+
+	/////////////////////
+	///  TEAM ROCKET  ///
+	/// Modifications ///
+	/////////////////////
+
+	private double launchAngle;
+
+    /**
```

Comment:
That's odd, in IntelliJ these are indented correctly... I will try auto-indent the whole file